src/app/server/Server.cpp

```diff
@@ -405,6 +415,9 @@ void InitServer(AppDelegate * delegate)
 
 // ESP32 and Mbed OS examples have a custom logic for enabling DNS-SD
 #if CHIP_DEVICE_CONFIG_ENABLE_MDNS && !CHIP_DEVICE_LAYER_TARGET_ESP32 && !CHIP_DEVICE_LAYER_TARGET_MBED
+    app::Mdns::SetSecuredPort(gSecuredServicePort);
```

Comment:
why are the ports not set for esp32 and mbed? 

I understand the comment of saying thse platforms have custom start code (I believe they use platform dns) however ports seem to not depend on this.

Reply:
fixed